services/prefect/flows/generated_flows.py

```diff
@@ -93,8 +95,58 @@ def create_flow_for_scraper(ix: int, d: Type[DatasetBase]):
     return flow
 
 
-for ix, cls in enumerate(ALL_SCRAPERS):
-    if not cls.autodag:
-        continue
-    flow = create_flow_for_scraper(ix, cls)
+def create_cdc_single_state_flow():
+    with Flow(CDCCovidDataTracker.__name__) as flow:
+        state = prefect.Parameter("state")
+        connstr = EnvVarSecret("COVID_DB_CONN_URI")
+        sentry_dsn = EnvVarSecret("SENTRY_DSN")
+        sentry_sdk_task = initialize_sentry(sentry_dsn)
+
+        d = create_scraper(CDCCovidDataTracker, state=state)
+        fetched = fetch(d)
+        normalized = normalize(d)
+        validated = validate(d)
+        done = put(d, connstr)
+
+        d.set_upstream(sentry_sdk_task)
+        normalized.set_upstream(fetched)
+        validated.set_upstream(normalized)
+        done.set_upstream(validated)
+
+    return flow
+
+
+def create_cdc_all_states_flow():
+    """Creates a flow that runs the CDC data update on all states."""
+    sched = CronSchedule("17 */4 * * *")
+
+    flow = Flow("CDCAllStatesDataUpdate", sched)
+    for state in ALL_STATES_PLUS_DC:
```

Comment:
Chris, I think this would start all states at the same time. 

Do you know if there is a way to do it batches, say start 5/wait, start next 5/wait, etc.?

Reply:
When I was running it locally it was running one after the other - so I don't think it should kick all off at once (I think the wait=True does that)

Reply:
ooh fancy -- I like that